### DISCUSSION TITLE
Uncaught SyntaxError: Unexpected token '}'

### DISCUSSION THREAD

**Discussion by heshamelmasry77** [alpinejs/alpine#2675] [1. Help]
I have this example but I get this bug any solution or reason why I get such an issue

```
@props( ['featured' => false] )

<li
    x-data="{
    isFeatured: {{ $featured }} }"
    class="col-span-1 flex flex-wrap shadow-sm rounded-md border border-gray-300 hover:border-gray-400 focus-within:ring-2 focus-within:ring-offset-2 focus-within:ring-gray-500">
```
![CleanShot 2022-02-15 at 17 23 50](https://user-images.githubusercontent.com/15607302/154104335-5f0eb015-6f09-4430-ae7f-2ea0c1112f6e.png)

**Comment by SimoTod** [comment]
check the generated html, it's probably invalid. Be aware that if you print `false` in php, you get nothing so probably your generated code looks like `x-data="{ isFeatured: }"` which is invalid javascript

**Reply by heshamelmasry77** [reply]
thank you for answering so I have solved it to just have it in a string 

```
@props( ['featured' => false] )

<li
    x-data="{ isFeatured: '{{ $featured }}' }"
```
